backend/app/platforms/schema.py

```diff
@@ -0,0 +1,11 @@
+from uuid import UUID
+
+from app.shared.schema import ORMModel
+
+
+class PlatformCreate(ORMModel):
```

Comment:
Do you expect to have Platform create API? E.g. for services, you have only PlatformService schema.